mcs/class/corlib/Mono/Runtime.cs

```diff
@@ -40,11 +40,26 @@ static class Runtime
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
 		private static extern void mono_runtime_install_handlers ();
 
-		static internal void InstallSignalHandlers ()
+#if MOBILE
+		public
+#else
+		internal
```

Comment:
This class is only visible in `MOBILE`, so that would have to be changed too.

But yeah, I think that would be a good idea.